components/resources/domain_block_strings.grdp

```diff
@@ -4,13 +4,13 @@
     Domain Blocked
   </message>
   <message name="IDS_DOMAIN_BLOCK_HEADING" desc="Heading of domain block interstitial page.">
-    Suspicious site ahead
+    The page ahead may attempt to track you across sites
   </message>
   <message name="IDS_DOMAIN_BLOCK_PRIMARY_PARAGRAPH" desc="Main paragraph of domain block interstitial page.">
     Brave has prevented the following site from loading:
   </message>
   <message name="IDS_DOMAIN_BLOCK_EXPLANATION" desc="Second paragraph of domain block interstitial page.">
-    This site may be fraudulent, bounce you between various sites to set tracking cookies, or perform other privacy invasive actions.
+    This page may be trying to link some of your personal data with a cookie before taking you to your intended destination. You can choose to allow this potential privacy leak and proceed to the next page, or you can block this tracking attempt by going back to the previous page.
```

Comment:
Let's use @karenkliu 's wording (see the screenshots instead of the GitHub comments) instead of what I proposed.

Reply:
Sorry, I should have commented to that effect on the issue.

Reply:
@fmarier the heading as well?